google/datalab/contrib/pipeline/airflow/airflow_vm_create.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+VM_NAME="instance-7"
+PROJECT_ID="cloud-ml-dev"
+ZONE="us-central1-b"
+
+# TODO(rajivpb): These should have more official values
+AIRFLOW_VM_STARTUP_SCRIPT="gs://rajivpb-airflow-testing/airflow_vm_startup.sh"
+GCS_DAG_BUCKET_FOR_AIRFLOW="rajivpb-airflow-testing"
+GCS_DAG_PATH_FOR_AIRFLOW="/dags"
+
+# The default values here were grabbed from Pantheon's "command-line" button for creating a VM
+gcloud beta compute --project $PROJECT_ID instances create $VM_NAME \
+--zone $ZONE \
```

Comment:
Nit - Slightly more readable if these were indented IMO.

Reply:
This PR is not for submission and is only for review since this isn't fully working yet. Updated title accordingly.